### PR TITLE
FIX Remove potentially overlapping site tree flags

### LINKED ISSUE #334: Fluent SiteTree labels overlay CMS labels
image

### PR DESCRIPTION
**Pull request by raissanorth**
Resolves #334 partially by picking up the UX suggestion made in https://projects.invisionapp.com/share/ZBFVZJCKU38#/screens/279729292_X_Fluent to remove the overlapping site tree flags

### CODE REVIEW

File: tests/php/Extension/FluentSiteTreeExtensionTest.php [hunk 1]
```diff
@@ -154,28 +154,13 @@ public function testFluentURLs($domain, $locale, $prefixDisabled, $pageName, $ur
         $this->assertEquals($url, $page->Link());
     }
 
-    public function testUpdateStatusFlagsInherited()
+    public function testUpdateStatusFlagsFluentInivisible()
```
Comment (robbieaverill) [inline tests/php/Extension/FluentSiteTreeExtensionTest.php]: Typo

File: tests/php/Extension/FluentSiteTreeExtensionTest.php [hunk 2]
```diff
@@ -154,28 +154,13 @@ public function testFluentURLs($domain, $locale, $prefixDisabled, $pageName, $ur
...
-        $flags = $page->getStatusFlags();
-
-        $this->assertTrue(array_key_exists('modified', $flags));
-
-        if (!array_key_exists('modified', $flags)) {
-            return;
-        }
-
-        $this->assertEquals('Locale drafted', $flags['modified']['text']);
+        $this->assertTrue(array_key_exists('fluentinvisible', $flags));
```
Comment (robbieaverill) [inline tests/php/Extension/FluentSiteTreeExtensionTest.php]: We can use `assertArrayHasKey` instead

File: tests/php/Extension/FluentSiteTreeExtensionTest.php (hunk 1 above)
Comment (raissanorth) [reply tests/php/Extension/FluentSiteTreeExtensionTest.php]: Oops! Well spotted.

File: tests/php/Extension/FluentSiteTreeExtensionTest.php (hunk 2 above)
Comment (raissanorth) [reply tests/php/Extension/FluentSiteTreeExtensionTest.php]: Sweet, have changed it.

**Pull request merged**